NYPLAudiobookToolkit/Bookmarks/NYPLAudiobookBookmarks.swift

```diff
@@ -0,0 +1,91 @@
+import Foundation
```

Comment:
another nitpicking request: could you rename `NYPLAudiobookBookmarks` to `NYPLAudiobookBookmark`  since it contains just the model class?